docs/services/valkey-connect.md

```diff
@@ -0,0 +1,52 @@
+# Connecting to a Valkey Server
+
+This document outlines how to connect to a Valkey server using two methods: 
+* `valkey-cli` command-line interface
+* Python with `redis-py` library
+
+## 0. Get the endpoint
+
+To get the endpoint to connect to, go to [Ivee portal](https://app.ivee.cloud) and in the "Instances" choose the instance. 
+In the "Connectivity" section you will find the Global endpoint and the Port.
```

Comment:
Do we have an example
You cannot use the "actual" endpoint or port because of security reasons, but you can use 
<endpoint>
<port number>

or something like that.

Reply:
Thanks! Fixed.